pkg/trace/agent/run.go

```diff
@@ -148,3 +140,17 @@ func Run(ctx context.Context) {
 		f.Close()
 	}
 }
+
+func initLogger(cfg *config.AgentConfig) {
+	if err := coreconfig.SetupLogger(
```

Comment:
Do we have to worry about this at all if there is a bad config with a bad log file in it?

Reply:
If there was a log file noted, it will go with it. If not - it will use a default path.

If configuration file cannot be opened/found/etc - it won't cause an error, defaults are applied